src/@3rdweb-sdk/react/hooks/useEngine.ts

```diff
@@ -33,6 +33,12 @@ export type EngineInstance = {
     | "deploymentFailed";
 };
 
+const getEngineRequestHeaders = (token: string | null): HeadersInit => ({
+  "Content-Type": "application/json",
+  "ngrok-skip-browser-warning": "true",
```

Comment:
Add the same comment here (or maybe just link the source: https://ngrok.com/abuse)